### PR TITLE
[16.0][MIG] mail_multicompany: Migration to 16.0

### PR DESCRIPTION
**Pull request by slitheryduke11**

### CODE REVIEW

**Comment by slitheryduke11** [comment]
Hi @luisg123v, @jesusVMayor, @Cedric-Pigeon, @ValentinVinagre. I think this migration is ready.

I would appreciate it if you could help me with the review.

Thanks in advance!

**Comment by ValentinVinagre** [comment]
Hello @slitheryduke11 ,
we do not support this module because when it was ported to OCA changes were made to which we did not agree. We use the one from our public repository.

All the best,

**Comment by slitheryduke11** [comment]
@ValentinVinagre, I understand.

Thank you for the information.

**Comment by desdelinux** [comment]
@dreispt @hugho-ad @moylop260 

could you review it please

**Review by dreispt — APPROVED** [review]

File: mail_multicompany/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Email Gateway Multi company",
+    "version": "16.0.0.1.0",
```
Comment (luisg123v) [inline mail_multicompany/__manifest__.py]: 16.0.1.0.0

File: mail_multicompany/tests/test_mail_multicompany.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        super().setUpClass()
+        cls.user_demo = cls.env.ref("base.user_demo")
+        company_obj = cls.env["res.company"]
+        server_obj = cls.env["ir.mail_server"]
+        cls.company1 = company_obj.create({"name": "Company 1"})
+        cls.company2 = company_obj.create({"name": "Company 2"})
+        cls.user_demo.write(
+            {
+                "company_id": cls.company1.id,
+                "company_ids": [(4, cls.company1.id), (4, cls.company2.id)],
```
Comment (luisg123v) [inline mail_multicompany/tests/test_mail_multicompany.py]: Use the Command syntax

File: mail_multicompany/models/mail_message.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+
+class MailMessage(models.Model):
+
+    _inherit = "mail.message"
+
+    company_id = fields.Many2one("res.company", "Company")
+
+    @api.model_create_multi
+    def create(self, values_list):
```
Comment (luisg123v) [inline mail_multicompany/models/mail_message.py]: `vals_list`, to preserve original signature

**Review by luisg123v — CHANGES_REQUESTED** [review]

**Comment by slitheryduke11** [comment]
Hi, @luisg123v. All suggestions done.

**Review by luisg123v — APPROVED** [review]
LGTM :+1: 

Hi @dreispt or @moylop260,

Could you merge, please?

**Pull request merged**